ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -136,6 +138,26 @@ static ConfirmationFunction SimpleConfirmationFunction() {
   return cf;
 };
 
+// Check if op is supported by backend using is_supported API
+static Status IsSupportedByBackend(
+    const Node* node, const ng::runtime::Backend* op_backend,
+    std::map<std::string, std::set<shared_ptr<ng::Node>>>& TFtoNgraphOpMap,
+    bool& is_supported) {
+  is_supported = true;
+
+  auto ng_op = TFtoNgraphOpMap.at(node->type_string());
```

Comment:
Use find to locate if `node->type_string()` is in the map. if not return a bad status.

otherwise the `at` could fail ungracefully, and then a TF Status would not be bubbled up well.